src/tools/token.js

```diff
@@ -0,0 +1,23 @@
+const jwt = require('jsonwebtoken')
+
+class Token{
+    static seed = "mi-seed-secreto-para-el-sistema"
+    static caducidad = "30d"
+
+    static getJwtToken(usuario){
+        return jwt.sign({usuario}, this.seed, {expiresIn:this.caducidad})
+    }
+
+    static compararToken(token){
+        return new Promise((resolve, reject) => {
+            jwt.verify(token, this.seed, (err, decoded) => {
+                if(err) reject()
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/ComfyStream/ComfyStream-Backend/pullRequest?prid=7157741)
```suggestion
                if(err) {reject()}
```